backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -464,27 +464,29 @@ Common::Keymap *SdlGraphicsManager::getKeymap() {
 		keymap->addAction(act);
 	}
 
-	act = new Action("SCL+", _("Increase the scale factor"));
-	act->addDefaultInputMapping("C+A+PLUS");
-	act->addDefaultInputMapping("C+A+KP_PLUS");
-	act->setCustomBackendActionEvent(kActionIncreaseScaleFactor);
-	keymap->addAction(act);
+		act = new Action("SCL+", _("Increase the scale factor"));
```

Comment:
I think this extra tab is wrong here, no?

Reply:
You're right, this slipped due to a late edit (I moved the if clause further below to only catch the graphical scalers but not the scaling factors).

I fixed this directly on master with commit:
https://github.com/scummvm/scummvm/commit/8cef7b8b692b42a83ccafc54e34140c07c80c8d7